sunflower-land/sunflower-land
# [FEAT] Faction Introductions

# Description

1. Open the door to people once they have joined a faction
2. Show an intro text

<img width="557" alt="Screenshot 2024-07-01 at 3 30 13 PM" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/8baec199-9aff-4a26-bd83-ec2dcb02d247">

## How to test?

1. Start with no faction
2. Visit Kingdom, all doors should be closed
3. Join a faction
4. Door opens
5. Enter house
